src/kmedoids_ucb.cpp

```diff
@@ -12,6 +12,20 @@
 #include <unordered_map>
 #include <regex>
 //#include <sstream>
+#include <signal.h>
+
+/**
+ *  \brief Kills the process.
+ * 
+ * Check whether ctrl+c signal is received.
+ * If signal is received it kilss the process.
```

Comment:
nit: `kilss`
Also, move to 1 line if it fits within 80 char limit, otherwise OK